src/test/cypress/support/pageobjects/ModelingExerciseAssessmentEditor.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * A class which encapsulates UI selectors and actions for the Modeling Exercise Assessment editor
+ */
+export class ModelingExerciseAssessmentEditor {
+    openAssessmentForComponent(componentNumber: number) {
+        cy.get('.apollon-row').getSettled(`.sc-furvIG >> :nth-child(${componentNumber})`).children().eq(0).dblclick('top', { force: true });
+    }
+
+    assessComponent(points: number, feedback: string) {
+        cy.get('.sc-nVjpj > :nth-child(1) > :nth-child(2) > :nth-child(1) > :nth-child(2)').type(`${points}`);
+        cy.get('.sc-nVjpj > :nth-child(1) > :nth-child(3)').type(`${feedback}`);
+    }
+
+    addNewFeedback(points: number, feedback?: string) {
+        cy.get('.btn').contains('Add new Feedback').click();
+        cy.get('.col-lg-6 >>>> :nth-child(1) > :nth-child(2)').clear().type(points.toString());
+        if (feedback) {
+            cy.get('.col-lg-6 >>>> :nth-child(2) > :nth-child(2)').type(feedback);
+        }
+    }
+
+    saveExampleAssessment() {
+        cy.contains('Save Example Assessment').click();
+    }
```

Comment:
I would suggest removing/renaming/repurposing this function, or explicitly state in the documentation that this class is used for both pages.

Reply:
Removed it because it doesnt belong in that pageobject.